readme.md

```diff
@@ -187,7 +174,7 @@ console.log(document.querySelectorAll("h2").eq(1).html());
 console.log(document.querySelectorAll("h2")[1].html());
```

Comment:
I almost feel like this exercise could be shortened. Looking at it from a new student's perspective... it could be a bit overwhelming. Maybe just give ~1/2 of these things to run, and them give the rule of thumb: generally you'll use `.eq()` so that you continue to have a jQuery element.